src/main.ts

```diff
@@ -5,21 +5,14 @@ import throng from "throng";
 import { initializeSentry } from "config/sentry";
 import { isNodeProd } from "utils/is-node-env";
 import { getFrontendApp } from "./app";
-import { proxyLocalWSForDev } from "~/src/spa-proxy";
 
 const start = async () => {
 	initializeSentry();
 	const app: Express = getFrontendApp();
 	const port = Number(process.env.TUNNEL_PORT) || Number(process.env.PORT) || 8080;
-	const server = app.listen(port, () => {
+	app.listen(port, () => {
 		getLogger("frontend-app").info(`started at port ${port}`);
 	});
-
-	/**
-	 * Running Proxy for Web sockets for running SPA locally,
-	 * Only for Dev environments for hot reload
-	 */
-	proxyLocalWSForDev(server);
```

Comment:
This is removed as the react-scripts can directly direct user browser to connect websocker on provided port